gulpfile.js

```diff
@@ -58,6 +61,20 @@ function getBundleConfig() {
     return config;
 }
 
+function getTests(cb) {
+    var files = [];
+    var walker = walk.walk(testSettings.input, { followLinks: false });
+
+    walker.on('file', function(root, stat, next) {
+        files.push(path.join(root, stat.name));
```

Comment:
Sicne we removed the filter from the gulp task, can we add it back here? We do still need to filter out anything that's not a `spec.js` file.